src/runtime/crdt/crdt-collection.ts

```diff
@@ -304,9 +304,11 @@ export function simplifyFastForwardOp<T>(fastForwardOp: CollectionFastForwardOp<
  * there's more than one such actor, returns null.
  */
 function getSingleActorIncrement(oldVersion: VersionMap, newVersion: VersionMap): string | null {
-  if (Object.keys(oldVersion).length !== Object.keys(newVersion).length) {
+  const oldNumActors = Object.keys(oldVersion).length;
+  const newNumActors = Object.keys(newVersion).length;
+  if (newNumActors < oldNumActors || newNumActors > oldNumActors + 1) {
```

Comment:
why is this check necessary?
newNumActors < oldNumActors : this can never happen, we never remove actors from the versionmap
newNumActors > oldNumActors + 1 : if this is the case, it would result in incrementedActors.length>1 so null will be returned below

so I think this check can be removed

Reply:
Good call, done